diff --git a/node_modules/@react-native/gradle-plugin/settings.gradle.kts b/node_modules/@react-native/gradle-plugin/settings.gradle.kts
index 1aa88a1..e26ff93 100644
--- a/node_modules/@react-native/gradle-plugin/settings.gradle.kts
+++ b/node_modules/@react-native/gradle-plugin/settings.gradle.kts
@@ -13,7 +13,7 @@ pluginManagement {
   }
 }

-plugins { id("org.gradle.toolchains.foojay-resolver-convention").version("1.0.0") }
+// plugins { id("org.gradle.toolchains.foojay-resolver-convention").version("1.0.0") } // Patched: incompatible with Gradle 9.3.1

 include(
     ":react-native-gradle-plugin",
